src/main/java/org/stellar/sdk/LedgerEntryNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package org.stellar.sdk;
+
+public class LedgerEntryNotFoundException extends Exception {
```

Comment:
maybe a style guideline to add, for any public class at minimum, add a javadoc for the class level, and would be nice to have javadoc on any public methods.

Reply:
Fixed.